python/lsst/meas/astrom/multiindex.py

```diff
@@ -121,8 +121,7 @@ def read(self):
                 self.log.warn("Unable to get filename for astrometry index %s", fn)
                 continue
             self.log.debug('Reading index from multiindex file "%s"', fn)
-            if multiindex_add_index(self._mi, fn, INDEX_ONLY_LOAD_METADATA):
-                raise RuntimeError('Failed to read index from multiindex filename "%s"' % fn)
+            self._mi.addIndex(fn, False)
```

Comment:
Yes. It calls `MultiIndex.addIndex` (defined in `astrometry_net.cc`), which throws `lsst::pex::exceptions::RuntimeError` if the index cannot be read.